From e28893e3a4f317e2abce517b24c7707c760d1878 Mon Sep 17 00:00:00 2001
From: Vagrant Cascadian <vagrant@reproducible-builds.org>
Forwarded: https://github.com/plk/biber/issues/395
Date: Mon, 12 Jul 2021 15:46:27 +0000
Subject: [PATCH 1/2] doc/biber.tex: Do not generate document ID.

The document ID is typically a hash of various information about the
built document including build path and time of document build.
---
 doc/biber.tex | 1 +
 1 file changed, 1 insertion(+)

diff --git a/doc/biber.tex b/doc/biber.tex
index 0df48fad..02e03b6a 100644
--- a/doc/biber.tex
+++ b/doc/biber.tex
@@ -1,4 +1,5 @@
 \documentclass{ltxdockit}
+\pdfvariable suppressoptionalinfo \numexpr512\relax
 \usepackage{btxdockit}
 \usepackage[british]{babel}
 \usepackage[strict=true,autostyle=true]{csquotes}
-- 
2.32.0

